packages/react-native/template/README.md

```diff
@@ -0,0 +1,70 @@
+This is a [**React Native**](https://reactnative.dev) project bootstrapped with [`npx react-native@latest init`](https://github.com/react-native-community/cli/tree/main#creating-a-new-react-native-project).
+
+# Getting Started
+
+>**Note:** Make sure you have completed the [React Native - Environment Setup](https://reactnative.dev/docs/environment-setup) instructions uptill [Creating a new application](https://reactnative.dev/docs/environment-setup#:~:text=Creating%20a%20new%20application) step before proceeding.
+
+## Step 1: Start the Metro Server
+
+To start Metro, run `npx react-native start` inside your React Native project folder:
+
+```bash
+npx react-native start
+```
+
+`react-native start` starts **Metro Bundler**.
+
+>**Note:** If you use the **Yarn** package manager, you can use `yarn` instead of `npx` when running React Native commands inside an existing project.
+
+## Step 2: Start your application
+
+Let Metro Bundler run in its own terminal. Open a _new_ terminal inside your React Native project folder. Run the following:
+
+```bash
+npx react-native run-android
```

Comment:
Can we add commands for both Android & iOS?

Reply:
Ohh! You're so quick! I was typing comment down 😅

Definitely.

Reply:
can we change the commands to be `yarn android` and `yarn ios`? Once the project is built, we really should steer away from anything npx

Reply:
IIRC, CLI internally checks and uses YARN by default, if present.

Sure, though I would still suggest to keep both `npx` and `yarn`. As, an absolute beginner may be unaware of the `yarn`.
WDYT?

Reply:
precisely because it might be tailored to absolute beginners, we should steer away as much as we can from something as unreliable as npx.

Reply:
Sure. That's a valid point.
May be we should reflect same on the website as well.

Reply:
yeah absolutely. Are the docs still using `npx react-native run-android`? we should also change those

Reply:
The webiste uses `npx` everywhere https://reactnative.dev/docs/next/environment-setup and a small note about `yarn`

May be this needs more discussion @cortinico ??

Reply:
either `npm run` or `yarn` are equivalent, `npx` is not. I would prefer `yarn` as you can just call `yarn android` and is easy to remember

Reply:
if you could setup a PR for the website me and Nico can help ensuring it lands 👍

Reply:
I'll try to open PR for website asap... 👍